src/lib/drive.ts

```diff
@@ -7,11 +7,13 @@ import fileDownload from 'js-file-download';
 
 import { ALEPH_CHANNEL } from 'config/constants';
 
-import type { AggregateType, IPCContact, IPCFile, IPCFolder, ResponseType, UploadResponse } from 'types/types';
+import type { AggregateType, IPCContact, IPCFile, IPCFolder, ResponseType, UploadResponse , IPCProgram} from 'types/types';
 
 export const MONTH_MILLIS = 86400 * 30 * 1000;
 
 class Drive {
+	public programs: IPCProgram[];
```

Comment:
The Drive contains logic related to files & folders, you shouldn't have programs stored in it :wink: